remappings.txt

```diff
@@ -1,5 +1,9 @@
 @openzeppelin/contracts/=node_modules/@openzeppelin/contracts/
-@openzeppelin/contracts-upgradeable/=node_modules/@openzeppelin/contracts-upgradeable
-forge-std/=node_modules/forge-std/src
-world-id-state-bridge=node_modules/@worldcoin/world-id-state-bridge/src
-linea-contracts/=node_modules/linea-contracts/contracts
+@openzeppelin/contracts-upgradeable/=node_modules/@openzeppelin/contracts-upgradeable/
+forge-std/=node_modules/forge-std/src/
+world-id-state-bridge/=node_modules/@worldcoin/world-id-state-bridge/src/
+linea-contracts/=node_modules/linea-contracts/contracts/
+@prb/test/=node_modules/@prb/test/src/
+@eth-optimism/=node_modules/@eth-optimism/
+@worldcoin/=node_modules/@worldcoin/
+hardhat/=node_modules/hardhat/
```

Comment:
Do we really need Hardhat btw? I haven't found any Hardhat or HRE mentions in your tests yet.

Reply:
by bad took care of it @kfastov